test/cluster.sh

```diff
@@ -18,19 +18,39 @@
 
 source $(dirname $0)/../vendor/github.com/knative/test-infra/scripts/e2e-tests.sh
 
-# Location of istio for the test cluster
-readonly ISTIO_YAML=./third_party/istio-1.0.2/istio.yaml
+CLUSTER_SH_CREATED_MANIFESTS=0
 
-function create_istio() {
-  echo ">> Bringing up Istio"
-  kubectl apply -f ${ISTIO_YAML}
+# Create all manifests required to install Knative Serving.
+function create_manifests() {
+  # Don't generate twice.
+  (( CLUSTER_SH_CREATED_MANIFESTS )) && return 0
+  local YAML_LIST="$(mktemp)"
+  # Generate manifests, capture environment variables pointing to the YAML files.
+  local FULL_OUTPUT="$( \
+      source $(dirname $0)/../hack/generate-yamls.sh ${REPO_ROOT_DIR} ${YAML_LIST} ; \
```

Comment:
Trying to use this, it fails if I first call `initialize`:

```
./cluster.sh: line 31: ./../hack/generate-yamls.sh: No such file or directory
ERROR: Error generating manifests
```

If I `pwd` before this line, it's `/usr/local/google/home/jonjohnson/go/src/github.com/knative/serving`

I'm confused though, maybe I don't understand how `dirname` works since it seems like this should work..

Reply:
Let's take this offline, this is working fine in `e2e-tests.sh` so probably it's something specific to your script.